velox/expression/tests/SparkExpressionRunnerTest.cpp

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <folly/init/Init.h>
+#include <gflags/gflags.h>
+#include <gtest/gtest.h>
+#include "velox/common/base/Fs.h"
+#include "velox/expression/tests/ExpressionRunner.h"
+#include "velox/expression/tests/ExpressionVerifier.h"
+#include "velox/functions/sparksql/Register.h"
+#include "velox/vector/VectorSaver.h"
+
+using namespace facebook::velox;
+
+DEFINE_string(
```

Comment:
These are the same as the ones present in presto expression fuzzer if I am not mistaken, maybe we can move the two to a common place rather than have 2 instances of same thing ?